external/header.php

```diff
@@ -130,7 +130,9 @@ function () {
         $time = array_key_exists('REQUEST_TIME', $_SERVER)
             ? $_SERVER['REQUEST_TIME']
             : time();
-        $requestTimeFloat = explode('.', $_SERVER['REQUEST_TIME_FLOAT']);
+        // In some cases there is comma instead of dot
+        $delimiter = (strpos($_SERVER['REQUEST_TIME_FLOAT'], ',') !== false) ? ',' : '.';
```

Comment:
Is your locale somewhere in europe? Some locales change how the string cast works for floats. You can check this with:

```bash
php -a
php > echo 3.14159;
3.14159
```